### DISCUSSION TITLE
Creating a Connection via REST API - correctly handling extras

### DISCUSSION THREAD

**Discussion by andersbogsnes** [apache/airflow#31962] [Q&A] [answered]
I'm trying to get the correct format for posting a Kubernetes Connection to the REST endpoint - the endpoint expects a string for the "extra" field, but I can't work out how to stringify properly and also have Airflow read the variable correctly. 

Is there any documentation on how this should be done?

**Comment by potiuk** [comment, marked as answer]
Yes. https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#generating-a-connection-uri

**Reply by andersbogsnes** [reply]
Thanks! Took a while to have time to verify, but that is working 👍

**Reply by potiuk** [reply]
Glad I could help
